auth/routes/routes.ts

```diff
@@ -22,6 +23,54 @@ router.get('/sesion', Auth.authenticate(), (req, res) => {
     res.json((req as any).user);
 });
 
+/**
+ * Listado de organizaciones a las que el usuario tiene permiso desde un modulo en particular.
+ * Momentaneamente solo se resuelve para Bi-Queries.
+ * @get /api/auth/bi-queries/organizaciones
+ */
+
+router.get('/:idModule/organizaciones', Auth.authenticate(), async (req: any, res, next) => {
+    let pipelineModulo = [];
+    pipelineModulo = [
+        {
+            $match: { 'submodulos._id': mongoose.Types.ObjectId(req.params.idModule) }
+        },
+        {
+            $unwind: '$submodulos'
+        },
+        {
+            $match: { 'submodulos._id': mongoose.Types.ObjectId(req.params.idModule) }
+        },
+        {
+            $project: {
+                _id: 0,
+                nombre: '$submodulos.nombre'
+            }
+        }
+    ];
+    const nombreModulo: any = await Modulos.aggregate(pipelineModulo);
+    const username = (req as any).user.usuario.username || (req as any).user.usuario;
```

Comment:
El req ya esta definido como any en la declaración del parametro, no hace falta forzarle el tipo
```suggestion
    const username = req.user.usuario.username || req.user.usuario;
```

Reply:
Corregido!